iFixit/react-commerce
# Add subscribe to newsletter collection section

This PR adds a subscribe to newsletter section to the collection page.

## QA

1. Visit the Vercel PR preview
2. Visit any collection page (other than `/parts` and `/tools`)
3. Verify that the newsletter section is displayed and looks like the mockup (see below for reference):

    <img width="851" alt="Screen Shot 2021-06-03 at 20 04 25" src="https://user-images.githubusercontent.com/4640135/120691548-e6c1ae00-c4a6-11eb-9733-334cf7d3732d.png">


4. Verify that the section is responsive
5. Try to subscribe an email, and verify that email is successfully subscribed ( cc @zdmitchell on how to verify correct subscription ). We're using the production endpoint, so you may want to remove the email after the workflow has been verified.

Closes #12
